### DISCUSSION TITLE
IOSF Blogs

### DISCUSSION THREAD

**Discussion by GouravSardana** [IndianOpenSourceFoundation/dynamic-cli#109] [General]
Hey Guys,

I hope you are doing well. Just curious to know - Do you guys want to share your journey in open source  Few points to add in the blog -

1. How you came to know about Indian Open Source Foundation? 
2. How you started contributing to Open Source?
3. What are the difficulties you have while contributing to Open Source?

You guys can use any platform to write blogs like medium, dev.to etc.

cc @pranavbaburaj @anshsrtv @GaneshRI @Ak-Shaw @sakshatl @guptaaastha @NamamiShanker (If I missed some of you. Please do comment down below. Really excited to talk to you guys.)

**Comment by NamamiShanker** [comment]
Hey Gourav! Thanks for starting this amazing project. I don't write blogs actually, but if I plan to, I'll share about IOSF for sure, as my contribution journey started from this project only.

**Reply by GouravSardana** [reply]
No worries. No one is professional blogger here. Just share your journey. I'll happy to share every blog from IOSF Linkedin account ;)

**Comment by guptaaastha** [comment]
Hi Gaurav,That's a good idea; I'll write an article on this topic in this month sometime and will share it over here. Glad to be a part of IOSF 😄

**Reply by GouravSardana** [reply]
Thanks @guptaaastha . Please do share when you have time to write some blog.

If you have any other person who is interested in open source. Please recommend. Will definitely help them !

**Comment by GaneshRI** [comment]
Most important problem as a freshman in most of colleges they don't know
about opensource and literally if they want to contribute to it, there is
no particular website that guides us.  Hopefully we have iosf now and
obviously Gsoc plays important role in this but first u have to get
experience in certain area to get opensource Internship. But if u really
want to contribute to open source except Gsoc it's really difficult to find
one. Now I know how to do it after some research and web crawling.

**Reply by GouravSardana** [reply]
You can write a blog about how to apply to those open source internship and how you crawl the data

**Comment by YourAKShaw** [comment]
Hi @GouravSardana, it's been a while hopping on a DM. 

TBH, I haven't made any contributions towards IOSF. But I may be able to throw some light in these questions:
- How you started contributing to Open Source?
- What are the difficulties you have while contributing to Open Source?

Since, I've been a maintainer for this [organisation](https://github.com/Crio-Bytes) during HacktoberFest last year, where contributors contributed mostly documentation for early stage developers, and me and my team thoroughly reviewed the same. 

Along the while, I'll be able to add details of how I came to know about **IOSF** and give details of some of the fantastic sessions we had.

Do let me know if that works.

**Reply by GouravSardana** [reply]
Yes that would definitely works

**Reply by YourAKShaw** [reply]
Cool, then kindly look forward to it within a week or so. :)
